src/main/java/core/basesyntax/service/DailyReport.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class DailyReport {
+
+    public List<String> listOperation(Map<String, Integer> fruitMap) {
+        List<String> result;
+        try {
+            result = fruitMap.entrySet()
+                     .stream()
+                     .map(entry -> entry.getKey() + "," + entry.getValue())
+                     .collect(Collectors.toList());
+        } catch (NullPointerException e) {
+            throw new RuntimeException(" Storage is empty  ", e);
+        }
```

Comment:
same here, lets not just catch NullPointerException, but check for null with if statement